views/admin/index.php

```diff
@@ -76,6 +75,17 @@
           },
         ],
         [
+            'attribute' => 'auth_items',
+            'format' => 'html',
+            'value' => function ($data) {
+                return implode(', ', array_map(function ($ai) {
+                    return $ai->name;
+                }, $data->authItems));
+            },
+            'filter' => ArrayHelper::map(Yii::$app->db->createCommand('select name from auth_item')->queryAll(), 'name', 'name'),
```

Comment:
Don't do queries in views, this has several ways to break.
1. Auth manager might not use DB.
2. Auth manager might be configured to use a different DB.